### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by HelloUniverseNo31415**
Checklist:

- [ ] Work has preview url
- [x] Tests have been added for new functionality
- [x] Work has been reviewed by team
- [x] Work has been reviewed by @craigayre/@eclennett

### CODE REVIEW

File: src/components/ConfirmationBox/ConfirmationBox.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ConfirmationBox from './ConfirmationBox';
+
+configure({adapter: new Adapter()});
+
+it('calls the correct functions when clicked', () => {
```
Comment (eclennett) [inline src/components/ConfirmationBox/ConfirmationBox.test.js]: I would probably split this up, in most cases it is best to only have one expect per test. If yes fails here but no passes you will have to dig into it to find which is failing.
"calls onYes function when yes is clicked" or something

File: src/components/ConfirmationBox/ConfirmationBox.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	const testFuncYes = jest.fn();
+	const testFuncNo = jest.fn();
```
Comment (eclennett) [inline src/components/ConfirmationBox/ConfirmationBox.test.js]: nit pick (feel free to ignore)
this is like a mock so would be better names mockOnYes or just handleYes

File: src/components/WebcamCapture/WebcamCapture.test.js [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
-	const wrapper = shallow(<WebcamCaptureContainer />);
-	expect(wrapper).toHaveLength(1);
+it("Tells the user that webcam isn't found if not found", () => {
+	const wrapper = shallow(
+		<WebcamCapture cameraConnected={false} cameraRef={{current: null}} />
+	);
+	expect(wrapper.text()).toEqual('Cannot access camera');
+});
+
+it('Renders webcam with correct props if webcamconnected', () => {
```
Comment (eclennett) [inline src/components/WebcamCapture/WebcamCapture.test.js]: previous test is good -lets discuss this one though

File: src/components/ConfirmationBox/ConfirmationBox.test.js (hunk 1 above)
Comment (HelloUniverseNo31415) [reply src/components/ConfirmationBox/ConfirmationBox.test.js]: done

File: src/components/ConfirmationBox/ConfirmationBox.test.js (hunk 2 above)
Comment (HelloUniverseNo31415) [reply src/components/ConfirmationBox/ConfirmationBox.test.js]: changed

File: src/components/WebcamCapture/WebcamCapture.test.js (hunk 3 above)
Comment (HelloUniverseNo31415) [reply src/components/WebcamCapture/WebcamCapture.test.js]: Ive updated the test

File: src/components/WebcamCapture/WebcamCapture.test.js [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
+it('Renders webcam with screenshot format of jpeg', () => {
+	const wrapper = shallow(
+		<WebcamCapture cameraConnected={true} cameraRef={{test: 'value'}} />
+	);
+	expect(wrapper.find('Webcam').props().screenshotFormat).toEqual(
+		'image/jpeg'
+	);
+});
+
+//[screenshotFormat="image/jpeg"][screenshotWidth=224]')
```
Comment (eclennett) [inline src/components/WebcamCapture/WebcamCapture.test.js]: remove

File: src/components/ConfirmationBox/ConfirmationBox.js [hunk 5]
```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import labels from '../../utils/labels';
 
 const ConfirmationBox = props => {
 	return (
 		<div>
-			<div>{props.text}</div>
+			<div>{`Is this a ${labels[props.item]}?`}</div>
```
Comment (eclennett) [inline src/components/ConfirmationBox/ConfirmationBox.js]: can you test this line too actually...?
just a contians text is fine

File: src/components/WebcamCapture/WebcamCapture.test.js (hunk 4 above)
Comment (HelloUniverseNo31415) [reply src/components/WebcamCapture/WebcamCapture.test.js]: oops, yeah will do

File: src/components/ConfirmationBox/ConfirmationBox.js (hunk 5 above)
Comment (HelloUniverseNo31415) [reply src/components/ConfirmationBox/ConfirmationBox.js]: Ive added a text includes 'is this a' so it doesnt fire due to the text on the buttons

**Pull request merged**